Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -2788,11 +2812,13 @@ void MediaPlayerPrivateGStreamer::createGSTPlayBin(const URL& url)
     if (!m_player->isVideoPlayer())
         return;
 
+#if !USE(GSTREAMER_HOLEPUNCH)
```

Comment:
Why?

Reply:
@magomez will be able to answer better than me.

Reply:
Probably added at some point because it was causing a problem on some platform, and then was kept for every release. Feel free to remove it.

Reply:
Ah, got it. At that point there might not be a videoSink (if the platform uses autovideosink). Also, when using holepunch we don't take into account the size and orientation of the video, as the rectangle area is given only by the size of the video element, not the video frames. So when using holepunch we don't care about the video caps change.